API/fleece/Fleece.h

```diff
@@ -1238,6 +1239,67 @@ while (NULL != (value = FLDictIterator_GetValue(&iter))) {
     /** @} */
 
 
+    //////// FORMATTED VALUE BUILDER
+
+
+    /** \defgroup builder   Fleece Formatted Value Builder
+         @{
+        These functions use the `printf` idiom to make it convenient to create structured Fleece
+        values in memory with one call. They create or modify a `FLMutableArray` or `FLMutableDict
+        by reading the given format string and the following arguments.
+
+        The format string is basically JSON5, except that any value in it may be a printf-style
+        '%' specifier instead of a literal, in which case that value will be read from the next
+        argument. The supported format specifiers are:
+
+        - Boolean:           `%c` (cast the arg to `char` to avoid a compiler warning)
```

Comment:
or '%b'?

Reply:
We have to use standard formats recognized by the compiler, for the compile-time parameter checking to work. `%b` isn't standard.